File: THCS/2021_2022/DaNang_2122/Bai2/DemKyTu.py
# Nhập xâu ký tự từ bàn phím
S = input("Nhập xâu ký tự S: ")

# Đếm số lượng ký tự 'A'
countA = 0
for c in S:
    if c == 'A':
        countA += 1

# In ra kết quả
print(f"Số lượng ký tự 'A' trong xâu S là: {countA}")

File: THCS/2021_2022/DaNang_2122/Bai2/DemKyTu.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string S;
    cout << "Nhập xâu ký tự S: ";
    getline(cin, S);  // Nhập toàn bộ chuỗi ký tự, bao gồm khoảng trắng

    int countA = 0;
    // Duyệt qua từng ký tự trong chuỗi
    for (char c : S) {
        if (c == 'A') {
            countA++;
        }
    }

    // In kết quả
    cout << "Số lượng ký tự 'A' trong xâu S là: " << countA << endl;
    return 0;
}
